### ISSUE TITLE
有关群晖套件版迁移到docker版的问题

### ISSUE THREAD

**Issue by foreverlong** [devome/dockerfiles#9]
大佬好 首先感谢您的付出
我现在遇到了一个问题：我把套件版的BT_BACKUP复制到docker版的BT_BACKUP的目录下之后，通过日志我可以看到如下：
`<html>
<body>

2022-02-15 03:10:26 | stdout | (N) 2022-02-15T11:10:25 - 无法将 "越狱S01-S05.Prison.Break.2005-2017.1080p.Blu-ray.x265.AC3￡cXcY@FRDS" 从 "/volume2/video2/Drama/" 移动到 "/volume2/video2/Drama"。两条路径都指向同一位置。
-- | -- | --
2022-02-15 03:10:26 | stdout | (N) 2022-02-15T11:10:25 - 无法将 "开端.Reset.2022.S01.1080p.WEB-DL.50fps.H265.10bit.HDR.DDP-LeagueWEB" 从 "/volume2/video2/Drama/" 移动到 "/volume2/video2/Drama"。两条路径都指向同一位置。
2022-02-15 03:10:26 | stdout | (N) 2022-02-15T11:10:25 - '开端.Reset.2022.S01.1080p.WEB-DL.50fps.H265.10bit.HDR.DDP-LeagueWEB' 已恢复。
2022-02-15 03:10:26 | stdout | (N) 2022-02-15T11:10:25 - '越狱S01-S05.Prison.Break.2005-2017.1080p.Blu-ray.x265.AC3￡cXcY@FRDS' 已恢复。
2022-02-15 03:10:26 | stdout | (N) 2022-02-15T11:10:25 - 无法将 "結城るみな" 从 "/volume1/StudyData/" 移动到 "/volume1/StudyData"。两条路径都指向同一位置。
2022-02-15 03:10:26 | stdout | (N) 2022-02-15T11:10:25 - 无法将 "Red.Notice.2021.1080p.NF.WEB-DL.DDP5.1.Atmos.DV.HEVC-TEPES.mp4" 从 "/volume4/video/Movie/红色通缉令.Red.Notice.2021.1080p.NF.WEB-DL.DDP5.1.Atmos.DV.HEVC-TEPES/" 移动到 "/volume4/video/Movie/红色通缉令.Red.Notice.2021.1080p.NF.WEB-DL.DDP5.1.Atmos.DV.HEVC-TEPES"。两条路径都指向同一位置。
2022-02-15 03:10:26 | stdout | (N) 2022-02-15T11:10:25 - 无法将 "Hidden Man 2018 1080p BluRay DD+5.1 x264-Geek.mkv" 从 "/volume4/video/Movie/邪不压正/" 移动到 "/volume4/video/Movie/邪不压正"。两条路径都指向同一位置。
2022-02-15 03:10:26 | stdout | (N) 2022-02-15T11:10:25 - 'Red.Notice.2021.1080p.NF.WEB-DL.DDP5.1.Atmos.DV.HEVC-TEPES.mp4' 已恢复。
2022-02-15 03:10:26 | stdout | (N) 2022-02-15T11:10:25 - '結城るみな' 已恢复。

</body>
</html>`
我看到您说“无法移动”是由于qbittorrent自身的原因，我发现好像所有的“无法移动”后面都有“已恢复”，然而在8080的webui中一个种子都没有。当我尝试下载一个种子时，出现如下提示：
`<html>
<body>

2022-02-15 03:00:47 | stdout | (N) 2022-02-15T11:00:46 - 无法将 "新型冠状病毒感染的肺炎防控知识手册.docx" 从 "/volume1/StudyData/1" 移动到 "/volume1/StudyData/1"。两条路径都指向同一位置。
-- | -- | --
2022-02-15 03:00:44 | stdout | (N) 2022-02-15T11:00:43 - 无法将 "新型冠状病毒感染的肺炎防控知识手册.docx" 从 "/volume1/StudyData/1" 移动到 "/volume1/StudyData/1"。两条路径都指向同一位置。
2022-02-15 03:00:44 | stdout | (N) 2022-02-15T11:00:43 - '新型冠状病毒感染的肺炎防控知识手册.docx' 已添加到下载列表。

</body>
</html>`
但同样的无法在webui中显示有这个种子。
大佬能否帮忙解答一下呢？感谢！
PS：还有一个小小的问题需要请教：我之前的群晖套件版中的保存路径为：“/volume4/video/Movie”，那么我直接在群晖中将“video”这个文件夹映射为“/volume4/video”的操作是否正确呢？

**Comment by devome** [comment]
要想恢复数据，映射后容器内的路径必须和你之前的一致，如果是从套件转移的，那么映射路径和真实路径需要一样。

**Comment by foreverlong** [comment]
我之前使用的套件版，所以路径是“/volume4/video/Movie” 之后我在群晖的docker设置里面把“video“这个共享文件夹映射为”/volume4/video“  请问这个操作是否是可以的呢？
其中共享文件夹”video”的真实路径是“/volume4/video”

**Comment by foreverlong** [comment]
![image](https://user-images.githubusercontent.com/43675453/153987136-bda074bf-554a-4914-8267-482331695715.png)

**Comment by devome** [comment]
> 我之前使用的套件版，所以路径是“/volume4/video/Movie” 之后我在群晖的docker设置里面把“video“这个共享文件夹映射为”/volume4/video“  请问这个操作是否是可以的呢？
> 其中共享文件夹”video”的真实路径是“/volume4/video”

如果video文件夹的真实路径确实是/volume4/video的话，那么这么映射是正确的。

**Comment by devome** [comment]
你在原来套件中的webui中，是可以看到下载资料的真实保存路径的

**Comment by foreverlong** [comment]
是的 原来的套件版的webui中的路径就是“/volume4/video/Movie”这样的 在我映射之后，好像也就不用修改路径了

**Comment by foreverlong** [comment]
那请问您对webui中不显示种子有什么头绪么？

**Comment by devome** [comment]
看看日志文件中怎么说？正常的是xxxx种子已恢复。如果日志中没问题，那么就是浏览器缓存的问题了，清除浏览器缓存试试。

**Comment by devome** [comment]
日志在映射的data/logs下

**Comment by foreverlong** [comment]
是的 日志文件在这个issue的最上面 
确实是显示“已恢复”，但同时显示“无法将  XXX从 "/volume2/video2/Drama/" 移动到 "/volume2/video2/Drama"。两条路径都指向同一位置。”
同时我刚刚更换浏览器后也是没有显示
我的种子可能是比较多，有600+个，会不会是因为种子太多无法显示了呢？

**Comment by devome** [comment]
日志看起来没问题，那么无法移动的问题是4.4.0官方的bug。
一定要清除浏览器缓存，然后种子多要等一会。

**Comment by foreverlong** [comment]
好的 多谢您的解答 我等待一下试试

**Comment by foreverlong** [comment]
您好 我刚刚查找发现我一共有657个种子，但只有503个已恢复

**Comment by devome** [comment]
那你检查下没恢复的具体问题先。

**Comment by foreverlong** [comment]
没有恢复的种子在日志中完全没有显示，也没有“无法移动”的提示

**Comment by foreverlong** [comment]
会不会是种子太多达到qbittorrent的上限呢？

**Comment by devome** [comment]
> 没有恢复的种子在日志中完全没有显示，也没有“无法移动”的提示

有几种情况：
1. 路径设置错了；
2. 种子本身有问题；
3. 种子不存在；
4. 快速恢复文件不存在；
5. 快速恢复文件有问题。

qb做种没有上限。

**Comment by foreverlong** [comment]
好的 我尝试重新复制一份种子和快速恢复文件

**Comment by foreverlong** [comment]
我刚刚把所有种子和恢复文件重新复制了一份，在log中所有657个种子都“已恢复”
![image](https://user-images.githubusercontent.com/43675453/153990476-f2f33c34-ce8d-4a1a-ac9f-d29f2890a474.png)
但在webui中还是一个种子都没有orz

**Comment by devome** [comment]
一样的啊，每次重装需要清空浏览器缓存。

**Comment by foreverlong** [comment]
是的 我不光清除了缓存 还用别的从来没用过的电脑打开webui 也是空的

**Comment by devome** [comment]
这就很神奇了，一般清空后重启浏览器都没问题。日志没问题的话，做种肯定是没问题的。如果有使用备用webui，先还原成默认ui。

**Comment by foreverlong** [comment]
但我发现了有一点 就是在外面显示都是0
![image](https://user-images.githubusercontent.com/43675453/153991247-79c4c783-f2c0-4c49-a63a-643a63cdb958.png)
但“空闲”里面是有一个种子的：
![image](https://user-images.githubusercontent.com/43675453/153991275-c39b82e3-8f29-4b26-8cf6-352a6f5eecf1.png)
我将这个种子删掉后就会出现另一个种子，但每次只有一个种子，而且点击继续是没有反应的

**Comment by foreverlong** [comment]
我尝试使用安卓版的应用去连接qbittorrent 也是显示只有一个种子

**Comment by devome** [comment]
你这个太奇怪了，你运行导出文件清单那条命令试试看看导出的结果对不对？

**Comment by foreverlong** [comment]
![image](https://user-images.githubusercontent.com/43675453/153992694-eb0f7994-c97d-4825-b7a7-967c8a14b8d7.png)
生成了657个种子orz 但就是不显示

**Comment by devome** [comment]
> ![image](https://user-images.githubusercontent.com/43675453/153992694-eb0f7994-c97d-4825-b7a7-967c8a14b8d7.png)
> 生成了657个种子orz 但就是不显示

你这问题我也不知道是啥原因了...

**Comment by foreverlong** [comment]
好的。。。多谢大佬解答...我再试试....
